Shared/Extensions/StringExtensions.swift

```diff
@@ -154,4 +154,24 @@ extension String {
         
         return textEntered
     }
+    
+    // Taken from https://gist.github.com/pwightman/64c57076b89c5d7f8e8c
+    public var javaScriptEscapedString: String {
+        // Because JSON is not a subset of JavaScript, the LINE_SEPARATOR and PARAGRAPH_SEPARATOR unicode
+        // characters embedded in (valid) JSON will cause the webview's JavaScript parser to error. So we
+        // must encode them first. See here: http://timelessrepo.com/json-isnt-a-javascript-subset
+        // Also here: http://media.giphy.com/media/wloGlwOXKijy8/giphy.gif
+        let str = self.replacingOccurrences(of: "\u{2028}", with: "\\u2028")
+            .replacingOccurrences(of: "\u{2029}", with: "\\u2029")
+        // Because escaping JavaScript is a non-trivial task (https://github.com/johnezang/JSONKit/blob/master/JSONKit.m#L1423)
+        // we proceed to hax instead:
+        do {
+            let encoder = JSONEncoder()
+            let data = try encoder.encode([str])
+            let encodedString = String(decoding: data, as: UTF8.self)
+            return String(encodedString.dropLast().dropFirst())
+        } catch {
+            return self
```

Comment:
My guess is sec may have an issue with this, you should probably return `nil` instead

Reply:
agree